clients/drcachesim/tracer/tracer.cpp

```diff
@@ -1807,6 +1807,9 @@ instrument_clean_call(void *drcontext, instrlist_t *ilist, instr_t *where,
         MINSERT(ilist, where,
                 XINST_CREATE_sub(drcontext, opnd_create_reg(reg_mine),
                                  opnd_create_reg(reg_global)));
+#elif defined(RISCV64)
+        /* FIXME i#3544: Not implemented */
+        DR_ASSERT_MSG(false, "Not implemented on RISC-V");
```

Comment:
We needed to handle the RISCV64 case here because the `#else` part uses architecture specific macros? It may be that other code needs to be adjusted too. Maybe add a TODO to port to RISCV64 at the top-level comment in this file too.

Reply:
Yes. IIUC `#else` is essentially x86 logic (since it has `lea`).